code/_onclick/drag_drop.dm

```diff
@@ -86,6 +86,7 @@
 
 /obj/item
 	var/canMouseDown = FALSE
+	var/syndicate = FALSE // Does this item have syndicate only functionality via hud buttons? Needs to be in this scope to encompass all Chameleon items - Hopek
```

Comment:
I just realized this, this is nasty. Var bloat, theres no point creating a new var on such a top level for a dozen item